test/Common/Constants/TestEncryptionConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Bit.Test.Common.Constants;
+
+public static class TestEncryptionConstants
+{
+    // V1-style encrypted strings (AES-CBC-HMAC formats) accepted by validators
+    public const string V1EncryptedBase64 = "2.QmFzZTY0UGFydA==|QmFzZTY0UGFydA==|QmFzZTY0UGFydA==";
```

Comment:
❓ : could we try to come up with a better name here or add to the summary to indicate which types of request fields this is expected to be used in? I had to go look at the tests to understand how it is supposed to be used.